### ISSUE TITLE
RichNick returns SteamName and not player Nick which breaks DarkRP compatibility

### ISSUE THREAD

**Issue by Norway174** [Earu/EasyChat#91]
In this commit https://github.com/Earu/EasyChat/commit/e80bf9f3c6b6e4f7e9c4e48d4bc478ba62df620a all :Nick calls were changed to :RichNick.

DarkRP changes the players names in Nick, and by not using Nick, the chat will use players Steam names, and not their RP names.

![image](https://user-images.githubusercontent.com/4808657/180617606-806dd0df-6b92-42b9-a1db-b02a9f0ad30e.png)
For the purpose of this image, I set my RP name to "Test" with DarkRP.

But my name doesn't show up as Test in the Chat box when I speak.

**Comment by Earu** [comment]
Hi thank you for reporting that issue, I wasn't sure it was from the DarkRP game mode. I'm not actually sure how to handle this properly, there is a conflict of overwrites here. However I'll definitely be looking into it.

**Comment by Earu** [comment]
Fixed in https://github.com/Earu/EasyChat/commit/5209a9213f209b5c0757b16702de9448cbdd4151

**Comment by Norway174** [comment]
Hey,
Not sure what's causing it this time. But I just updated the EasyChat again from the Workshop.

And it seems the DarkRP chat names are broken again. It's showing the Steam names, instead of the RP names.
I'll try and help out some more and look into exactly which update is doing it. But sadly I don't have the time right now.

But I figured I'd re-open this issue again so you're aware of it. Or maybe you even already have an idea what could be causing it?
